src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -34,15 +34,45 @@ case class Face(normalVec: (Double,Double,Double), distance: Double, phi: Double
 
 object Block {
   /**
-   * Find a bounding sphere for a rock block.
-   * @param centerX The x coordinate of the block's center
-   * @param centerY The y coordinate of the block's center
-   * @param centerZ The z coordinate of the block's center
-   * @param faces A sequence of faces specifying the block's boundaries
-   * @return A pair where the first element is a triple giving the center of
-   *         the bounding sphere and the second element is the radius of the
-   *         bounding sphere.
-   */
+    * Calculates the rotation matrix to rotate the input plane (specified by its normal)
+    * to the desired orientation (specified by the desired normal)
+    * @param n_current: Current normal of plane
+    * @param n_desired: Desired new normal
+    * @return 3*3 rotation matrix
+    */
+  def rotationMatrix(n_current: (Double, Double, Double), n_desired: (Double, Double, Double)):
+                             DenseMatrix[Double] = {
+    val n_c = DenseVector[Double](n_current._1, n_current._2, n_current._3)
+    val n_d = DenseVector[Double](n_desired._1, n_desired._2, n_desired._3)
+    if (math.abs(linalg.norm(linalg.cross(n_c,n_d))) > NumericUtils.EPSILON) {
+      val v = linalg.cross(n_c, n_d)
+      val s = linalg.norm(v)
+      val c = n_c dot n_d
+
+      val v_skew = DenseMatrix.zeros[Double](3,3)
```

Comment:
I know I asked about this before, but can you explain why this is called `v_skew`? I understand the skew part, but why use the letter `v`, which to me usually indicates a vector rather than a matrix?